README.rst

```diff
@@ -35,7 +35,14 @@ Single murano services on the controller node
           user: openstack
           password: password
           virtual_host: '/openstack'
-          ha_queues: True
+        murano_agent:
```

Comment:
I don't like this, but change would be complicated and would raise unwanted questions, so ok :)